apb.yml

```diff
@@ -8,7 +8,7 @@ tags:
 metadata:
   displayName: Mobile Developer Console
   longDescription: "An APB deploying the AeroGear mdc-server Server with persistent storage and authentication through Openshift"
```

Comment:
can we update the description here to remove `AeroGear`, and change `mdc-server` to `Mobile Developer Console`

Reply:
can we also change the description in L3 as well, Just call it `Mobile Developer Console`

Reply:
actually, there isn't persistent storage here. might just remove it.

Reply:
👍 Done!